src/electron/add_tap_device.bat

```diff
@@ -17,6 +17,11 @@
 @echo off
 
 set DEVICE_NAME=outline-tap0
+set SERVICE_NAME=tap0901
+
+:: https://support.microsoft.com/en-us/help/556009
+reg Query "HKLM\Hardware\Description\System\CentralProcessor\0" | find /i "x86" > NUL && set OS=i386 || set OS=amd64
```

Comment:
The parameter to this script bugs me too. This is a neat way to do it but even so I'd prefer not scan the registry when we have an NSIS macro that will do it for us. You motivated me to finally address this here, WDYT?:
https://github.com/Jigsaw-Code/outline-client/pull/364

Reply:
I think that with a script that can solve the problem, don't spread to two scripts.

Reply:
An additional benefit of https://github.com/Jigsaw-Code/outline-client/pull/364 is that `tapinstall.exe` will appear in the same location on disk for 32 and 64 bit users. I just added that to the review thread - and to be clear, with that change there is no need for this script to check the system architecture.

Reply:
OK. I will close this thread. 

Another important thing. Please review the code for this zip file. It will solve a lot of issues.
[OutlineService.zip](https://github.com/Jigsaw-Code/outline-client/files/2499100/OutlineService.zip)